Titta.m

```diff
@@ -4,7 +4,7 @@
 % Titta can be found at https://github.com/dcnieho/Titta. Check there for
 % the latest version.
 % When using Titta, please cite the following paper:
-% Niehorster, D.C., Andersson, R. & Nystr�m, M., (in prep). Titta: A
+% Niehorster, D.C., Andersson, R. & Nystr�m, M., (in prep). Titta: A
```

Comment:
encoding conversion problem, not sure how I can stop it so you may need to fix it in your encoding mode

Reply:
The files are encoded in WINDOWS-1252 I think. I have been trying to convert them to utf8, but that does not yield anything committable, so i'm kinda at a loss here. On top of that, it seems that the matlab editor is not unicode-aware from my reading, so saving files there will mess it up anyway. I guess I'm simply replacing ö with o...
Do you have a better idea about this?

Reply:
No, the problem rests firmly with Mathworks inability to update to a UTF8 capable editor, so I just remove non-essential characters, but as I work in China, often colleagues comment in chinese and that breaks in my editor too. I normally just make small edits in VS Code where I can switch encoding as necessary. For this pull I didn't realise until after the commit...

The nerdy solution would be that git can [automate scripts](https://git-scm.com/docs/githooks) to perform encoding conversions automagically but I think it isn't worth the effort to develop that either...

Reply:
Actually you may be able to force the use of UTF-8 on Windows: 

www.mathworks.com/matlabcentral/answers/280988-how-do-i-get-my-matlab-editor-to-read-utf-8-characters-utf-8-characters-in-blank-squares-in-editors

On my mac, at least under 2019b utf-8 is the default now... You could batch convert all files using iconv or something similar.